_posts/2018-03-02-rails-to-aws-lambda-communication-and-authentication.md

```diff
@@ -87,7 +87,7 @@ Typically a session cookie (ie. one used for authentication) is created by the s
 <br/>
 ### Token
 
-[Tokens](https://jwt.io/introduction/){:target='_blank'} work in the same vein as cookies, except we skip the automated browser functionality which includes the cookie with every request. One reason to manually do what cookies do for free is the domain limitation of cookies. A session cookie can only be sent to the domain where it originated. So authentication data set by example.com cannot be sent to google.com. 
+[Tokens](https://jwt.io/introduction/){:target='_blank'} work in the same vein as cookies, except we skip the automated browser functionality which includes the cookie with every request. One reason to manually do what cookies do for free is the domain limitation of cookies. A session cookie can only be sent to the domain where it originated. So authentication data set by example⋅com cannot be sent to google⋅com. 
```

Comment:
Why do we need this change?

Reply:
I've no idea - it was mentioned in youtrack by Nadia, I assume they wanted to remove the user's ability to navigate to these websites from here which is why I've changed from a normal dot to a mid height dot? I can reverse this if you think it unnecessary